src/test/java/org/jenkinsci/plugins/gitclient/GitClientTest.java

```diff
@@ -258,6 +256,23 @@ public void setGitClient() throws IOException, InterruptedException {
         gitCmd.run("config", "user.email", "email.from.git.client@example.com");
     }
 
+    /**
+     * Allow local git clones to use the file:// protocol by setting
+     * protocol.file.allow=always on the git command line of the
+     * GitClient argument that is passed.
+     *
```

Comment:
```suggestion
     * <p>
```

for anyone actually reading as HTML